cli/create/builtin_templates/templates/cartridge/.cartridge.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Here you can specify default parameters for local running, such as:
+
+# cfg: path-to-cfg-file
+# log-dir: path-to-log-dir
+# run-dir: path-to-run-dir
+# data-dir: path-to-data-dir
+
+stateboard: true # If no arguments are supplied, --stateboard flag is true
```

Comment:
What does the `stateboard` flag mean?
We start the application by `tt start` command, not `cartridge start`.